src/view/console/output.js

```diff
@@ -0,0 +1,27 @@
+import { OUTPUT_MESSAGE, PRIZE_MESSAGE } from "../../constants/message";
+
+export const output = {
+  lottosCount(count) {
+    console.log(OUTPUT_MESSAGE.PURCHASE_RESULT_COUNT(count));
+  },
+
+  lottos(lottos) {
+    lottos.map((lotto) => console.log(lotto));
+  },
```

Comment:
마이너하지만 이렇게도 처리하면 좀 깔끔해집니다!

```suggestion
  lottos(lottos) {
    lottos.map(console.log);
  },
```